superset/connectors/sqla/models.py

```diff
@@ -35,6 +35,7 @@
     DateTime,
     desc,
     ForeignKey,
+    func,
```

Comment:
I'm somewhat perplexed why the imports have changed on this file as they don't seem to be used. Maybe this branch should be rebased against `master`.